### DISCUSSION TITLE
Show node that causes error due to data containing the ":" during pydot export

### DISCUSSION THREAD

**Discussion by KelvinChung2000** [networkx/networkx#7546] [Ideas]
Exporting to pydot will fail when the node string contains ":". We should show which node and potentially which field got that "erroneous" data. This will make debugging much more straightforward. 

I am happy to open a pull request and implement this since it is straightforward.

**Comment by MridulS** [comment]
I think that would be helpful, pointing out exactly what failed.

**Reply by KelvinChung2000** [reply]
I will make a pull request when is done.
